notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex.ipynb

```diff
@@ -20,14 +20,17 @@
   {
```

Comment:
<p>Can you add the bash commands as an example in markdown instead of removing it completely?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/702/discussion/#comment-2430744585'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>